dwitter/templatetags/insert_magic_links.py

```diff
@@ -44,7 +44,8 @@ def insert_magic_links(text):
     )
     text = re.sub(
         r'(?P<text>'                                            # capture original pattern
-        r'#(?P<hashtag>[_a-zA-Z][_a-zA-Z\d]*)[^_a-zA-Z\d]?)',   # hashtag
+        # hashtag (that isn't preceded with a : so web anchors aren't considered)
+        r'(?:^|\s)(?=\S*#)(?!\S*:)\S*#(?P<hashtag>[_a-zA-Z][_a-zA-Z\d]*))',
```

Comment:
Would be worth adding the explanation from the PR as a comment here :)

Reply:
Done!